language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/builder/FieldCompletionItemBuilder.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.langserver.completions.builder;
+
+import io.ballerina.compiler.api.symbols.ObjectFieldSymbol;
+import io.ballerina.compiler.api.symbols.RecordFieldSymbol;
+import io.ballerina.compiler.api.symbols.Symbol;
+import org.eclipse.lsp4j.CompletionItem;
+
+/**
+ * Completion item builder for the object fields and for record fields.
+ *
+ * @since 2.0.o
+ */
+public class FieldCompletionItemBuilder {
+    private FieldCompletionItemBuilder() {
+    }
+
+    /**
+     * Build the constant {@link CompletionItem}.
+     *
+     * @param symbol record field symbol or the object field symbol
+     * @return {@link CompletionItem} generated completion item
+     */
+    private static CompletionItem getCompletionItem(Symbol symbol) {
+        String recordFieldName = symbol.getName().orElseThrow();
+
+        CompletionItem completionItem = new CompletionItem();
+        completionItem.setLabel(recordFieldName);
+        completionItem.setInsertText(recordFieldName);
```

Comment:
Can we set the the `completionItem.kind()` as well here? For record fields, it's `Struct` and for object fields it should be `Interface` We have used the same kinds at `org.ballerinalang.langserver.completions.builder.TypeCompletionItemBuilder#setMeta()`

Reply:
@IMS94 thank you, I think we have missed setting the kind from the beginning. One correction here though, we have to set the kind as `Field`. We have used `Struct` for record variables and `Interface` for the Object variables. Hence I'll be setting the kind as `Field` here. Addressed with  607ade185fe7a8128e80b1a9693de3a175d5face

Reply:
@nadeeshaan I see. Great!